src/main/resources/org/jenkinsci/plugins/reverse_proxy_auth/ReverseProxySecurityRealm/config.jelly

```diff
@@ -40,8 +40,7 @@ THE SOFTWARE.
       <f:textbox name="customLogOutUrl" value="${instance.customLogOutUrl}" />
     </f:entry>
 	<f:entry title="${%Server}" >
-	  <f:textbox name="server" value="${instance.server}"
-            checkUrl="'${rootURL}/securityRealms/LDAPSecurityRealm/serverCheck?field=server&amp;server='+encodeURIComponent(this.value)+'&amp;managerDN='+encodeURIComponent(this.form.elements['managerDN'].value)+'&amp;managerPassword='+encodeURIComponent(this.form.elements['managerPassword'].value)"/>
```

Comment:
This looks like form validation is removed. Deliberate?

Reply:
I have removed the check since it was no longer working and throwing a 404. I will update the PR description.

It is first (and only) mentioned in [JENKINS-156](https://issues.jenkins.io/browse/JENKINS-36156), which points to [a google group question which you answered](https://groups.google.com/g/jenkinsci-users/c/HBHxN8SG2lw), adding that the plugin should not rely on internal URLs.

The current LDAP plugin has a `doValidate` endpoint with https://github.com/jenkinsci/ldap-plugin/blob/e5925c2efe39c4181b7ac77d2ebbb4f78ac7a856/src/main/java/hudson/security/LDAPSecurityRealm.java#L1531
which this plugin shouldn't be using either, I guess.

Reply:
> [a google group question which you answered](https://groups.google.com/g/jenkinsci-users/c/HBHxN8SG2lw)

Come on, that was over six years ago! 😄